uobtheatre/productions/schema.py

```diff
@@ -53,6 +58,18 @@ class Meta:
         interfaces = (relay.Node,)
 
 
+class ProductionFilter(django_filters.FilterSet):
+
+    start = django_filters.DateTimeFilter(method="start_filter")
+    id = GlobalIDFilter()
```

Comment:
The secret sauce. Might be worth creating a custom FilterSet class that adds this automatically because we will always want to be able to filter on id